Gopkg.toml

```diff
@@ -36,6 +37,10 @@ required = [
   name = "github.com/securego/gosec"
   version = "1.3.0"
 
+[[constraint]]
+  name = "honnef.co/go/tools"
+  version = "=2019.1.1"
```

Comment:
Could you comment on why we need to pin this? I prefer similar tools to be unpinned so that we can get the latest checks without manually bumping up.

Reply:
Primarily I pinned it because the project is using an odd versioning scheme, which is not a semver. So even if we don't pint it, it would be treated as ^2019.x.x, i.e. will not upgrade next year. But I can unpin, since we don't have an API dependency.